includes/utilities.inc

```diff
@@ -364,7 +364,7 @@ function islandora_solr_prepare_solr_results($solr_results) {
       }
 
       // Implode.
-      $value = is_array($value) ? implode(", ", $value) : $value;
+      $value = is_array($value) ? implode(variable_get('islandora_solr_search_field_value_separator', ', '), $value) : $value;
```

Comment:
@whikloj if we implode here, we can't implode again at the display layer with a different separator.